appveyor.yml

```diff
@@ -26,6 +26,15 @@
     - cmd: >-
         subst s: c:\projects
     - ps: set-location -path s:\node_modules\node-sass
+    - ps: Uninstall-Product node 4 $env:platform
+    - ps: Write-Host "Installing node-chakracore..." -ForegroundColor Cyan
+    - ps: Write-Host "Downloading..."
+    - ps: $msiPath = "$env:USERPROFILE\node-chakracore-$env:platform.msi"
+    - ps: $file = "https://github.com/nodejs/node-chakracore/releases/download/node-chakracore-7.0.0-pre9/node-chakracore-$($env:platform).msi"
```

Comment:
> We will not release official release binaries

Could you please elaborate on this? Do you mean that `binding.node` built using `node-chakracore` will not get hosted on https://github.com/sass/node-sass/releases ?

> until node-chakracore can a offer stable ABI

You mean until node-chakracore binaries are not available on CI system of nodejs.org, right?